### DISCUSSION TITLE
Maneira correta de realizar um insert em uma tabela associativa.

### DISCUSSION THREAD

**Discussion by iiChina** [balta-io/forum#75] [Tira Dúvidas] [answered]
Acesso à dados com .NET, C#, Dapper e SQL Server.

Ao final do módulo 4 foi proposto o desafio de criar vínculos entre tabelas com relacionamento muitos pra muitos, mas fiquei com dúvida sobre como fazer isso de uma forma "elegante". 

Nesses casos é necessário a criação de uma classe que represente essa tabela associativa no código?

**Comment by andrebaltieri** [comment, marked as answer]
@iiChina depende do modelo que está trabalhando. Se estiver em um cenário orientado à dados, a maneira mais elegante é esta. Se estiver utlizando um outro ORM como EF, provavelmente ele fará o Track das entidades e incluirá automaticamente.

Vale a pena conferir o curso de EF (Próximo da carreira) 💜

**Reply by iiChina** [reply]
Entendi, vou olhar sim! Obrigado Balta.
